docs/en/latest/plugins/http-logger.md

```diff
@@ -95,16 +95,6 @@ hello, world
 
  Note that **the metadata configuration is applied in global scope**, which means it will take effect on all Route or Service which use http-logger plugin.
 
-**APISIX Variables**
-
-|   Variable Name  |      Description        | Usage Example  |
-|------------------|-------------------------|----------------|
-| route_id         | id of `route`          | $route_id      |
-| route_name       | name of `route`        | $route_name    |
-| service_id       | id of `service`        | $service_id    |
-| service_name     | name of `service`      | $service_name  |
-| consumer_name    | username of `consumer` | $consumer_name |
-
```

Comment:
As we are deleting the table, I think it would be better to add a hyperlink to the newly created `apisix-variable.md` inside the metadata schema `log_format` description [L94] (Also same to all the similar updates).

Reply:
Done